### ISSUE TITLE
Don't insert a class into sys.modules

### ISSUE THREAD

**Issue by WeaverThree** [amreuland/PyVDF#1]
The code in your __init__.py does this. That means that if you try to import PyVDF in two different files that get loaded in the same interpreter one of them will fail because python doesn't know what the heck to do with what it finds in sys.modules.

Contrived example:

```
>>> import PyVDF.core
>>> import PyVDF.core as A
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object 'PyVDF' has no attribute 'core'
>>>
```

**Comment by noriah** [comment]
Alright. I'll fix this soon. Thanks for bringing this up. I was very new to python when I wrote this.

**Comment by noriah** [comment]
Very sorry for the long delay. I fell sick and was unable to do much after missing so much class. I will fix this tonight.

**Comment by noriah** [comment]
Fixed with 3d546178153204a1286c61be8f641b59f6c2750e
